frontend/testing/playwright/helpers/RouterRoute.ts

```diff
@@ -33,6 +33,10 @@ export class RouterRoute extends StudioEnvironment {
     return routerRoute;
   }
 
+  public getGiteaRoute(appName: string): string {
+    return `/repos/${this.org}/${appName}`;
```

Comment:
I don't think we need to implement a custom method for getting the gitea route. We should be able to add it to the `routerRoutes` object on line 13?

Reply:
Can do that yes 😄 
Thought that object was only for routes inside "Altinn Studio"